lib/beacon/authorization/behavior.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Beacon.Authorization.Behaviour do
```

Comment:
Hey @xn I was thinking about that callback for the pages query that we discussed yesterday. What if we don't have provide such callback for now? The `authorized?` callback will limit who can edit or delete each page, which I believe is enough for now, and making the list of pages public for all users is okay. I feel that adding the query callback complexity isn't worth the value it will bring, unless I'm missing something. Wdyt?

Reply:
We definitely don't have to do so now.